themes/origins/styles/component-slider.scss

```
@@ -81,7 +120,12 @@ yc-slider {
       button {
         padding: 8px 15px;
         border-radius: var(--radius-sm);
-        background-color: var(--color-secondary);
+        background-color: rgb(var(--section-arrows-background));
+        color: var(--section-arrows-foreground);
+
+        &.icon:hover {
+          background-color: rgba(var(--section-arrows-background), 0.5);
+        }
```

`0.7` is fair since `0.5` is already used for the disabled state

```suggestion
        &.icon:hover {
          background-color: rgba(var(--section-arrows-background), 0.7);
        }
```

---

Used color-mix instead :pepeok: